.env.kintsugi

```diff
@@ -1,5 +1,5 @@
 REACT_APP_BITCOIN_NETWORK="mainnet"
 REACT_APP_PARACHAIN_URL="wss://api-kusama.interlay.io/parachain"
 REACT_APP_STATS_SERVER_URL="https://api-kusama.interlay.io/stats"
-REACT_APP_HYDRA_URL=https://api-kusama.interlay.io/graphql/graphql
+REACT_APP_HYDRA_URL="https://api-kusama.interlay.io/graphql/graphql"
```

Comment:
For consistency and avoiding any potential mistake, I wrapped the link in quotes.